internal/watcher/watcher.go

```diff
@@ -169,15 +161,16 @@ func (d Differential) sendDiff(path string, diff string) error {
 	return nil
 }
 
+// WatchFn for reading files in the watched folder and look for difference from the last update
 func (d Differential) WatchFn(path string) error {
-	if _, ok := d.ignoredExtensions[filepath.Ext(path)]; ok {
+	if _, ok := d.allowedExtensions[filepath.Ext(path)]; !ok {
 		return nil
 	}
 
 	// This is to allow for changes to propagate on the filesystem. If the file
 	// is large, the write won't be atomic. It will happen in, for example, 4096
 	// bytes chunks. 1 ms should be enough.
-	time.Sleep(time.Millisecond)
+	time.Sleep(10 * time.Millisecond)
```

Comment:
Why was this changed?

Reply:
@hanzei the original code in `master` branch uses 1 ms. 10 ms is from PR's previous changes.

Reply:
Reverted by https://github.com/mattermost/mattermost-plugin-nagios/pull/31/commits/6761c161991e6304f20e77fadd6712ba758b28b7